assume/world.py

```diff
@@ -584,12 +644,16 @@ def add_market(self, market_operator_id: str, market_config: MarketConfig) -> No
         self.markets[f"{market_config.market_id}"] = market_config
 
     async def _step(self):
+        if self.distributed_role:
```

Comment:
please add documentation as docsting to this function. It is very hard to understand and won't be understood by anyone except for the core developers